dotfiles/bashrc

```diff
@@ -70,26 +93,22 @@ alias t='tmux attach-session -t "$(whoami)" || tmux new-session -s "$(whoami)"'
 # update all Vim plugins
 alias vundle='vim +PluginInstall! +qall'
 
+# DEV ENVIRONMENT =============================================================
 
-# FUNCTIONS ===================================================================
-
-# creates a directory and then immediately moves you to that directory
-take() {
-  mkdir -p $1;
-  cd $1
-}
+# Android SDK
+if [ -d "$HOME/Library/Android/sdk" ]; then
+  export ANDROID_HOME="$HOME/Library/Android/sdk"
+  export PATH=$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools:$PATH
+fi
 
-# if you name your project folder "FOO" when your production heroku
-# application is named "FOO-production", then you can call:
-#
-#     prod heroku [cmd] [opts]
-#
-function prod() {
-  $* --app $(basename $(pwd))-production
-}
+# Node Version Manager (NVM)
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # loads nvm
 
+# Ruby Version Manager (RVM)
+[[ -s "$HOME/.rvm/scripts/rvm" ]] && . "$HOME/.rvm/scripts/rvm" # loads rvm
```

Comment:
This is covered on https://github.com/bendyworks/buffet/blob/8a4bd41ef1204953dab9eaaa350dae0ba0bf5a15/dotfiles/bashrc#L188

which, iirc, is near the end so RVM doesn't complain about not being the last thing that's sourced or some other little detail like that that